python/helpers/requirements.txt

```diff
@@ -1,5 +1,5 @@
-pip>=21.3.1  # Allow earlier versions to retain python 3.6 support
-pip-tools>=6.4.0  # Allow earlier versions to retain python 3.6 support
+pip>=21.3.1,<=22.1.1  # Allow earlier versions to retain python 3.6 support
+pip-tools>=6.4.0,<=6.6.2  # Allow earlier versions to retain python 3.6 support
```

Comment:
This was mostly to break docker build cache so it would get the latest version of pip-tools, but also this will cause dependabot to manage these dependencies again.  Either way, I'm happy to revert this change as it was just to force a rebuild of the helpers.

Reply:
> also this will cause dependabot to manage these dependencies again. 

Pinning makes sense to me for this reason alone, although I'd probably pin a single concrete version rather than a range just for more determinism.

That said, I'm fine either way.

Reply:
Yeah, the ranges are here again to support earlier python versions.  The latest pip and pip-tools have removed support for python 3.6, so if we just run latest, we can't support 3.6.  We could pin to the earlier version, but then we have to make sure not to merge any dependabot PRs which will try to pull the pin forward...

Reply:
Oh right, that makes complete sense to keep the range.